lib/iex/lib/iex/introspection.ex

```diff
@@ -110,7 +110,12 @@ defmodule IEx.Introspection do
             || find_default_doc(docs, fun, arity)
 
       if doc do
-        print_doc(doc)
+        if match?({_, _, _, _, nil}, doc) && (behaviour_module = callback_module(mod, fun, arity)) do
```

Comment:
It feels unclear transition of names `behaviour_module` –> `callback_module`. Can both be `callback_module`?